blocks/extras/initialization.py

```diff
@@ -34,3 +34,19 @@ def make_matrix(size, perm):
             return make_matrix(shape[0], self.permutation)
         else:
             return make_matrix(shape[0], rng.permutation(shape[0]))
+
+
+class NormalizedInitialization(NdarrayInitialization):
+    u"""Initialize the parameters with the Glorot method, as introduced in
```

Comment:
There is no scrutinizer in this repo, but the first line should be a short description. Your description doesn't fit in one line, so you'll have to add a bit more detailed paragraph after a blank line.